agent/engine/daemonmanager/daemon_manager_linux.go

```diff
@@ -79,16 +90,23 @@ func (dm *daemonManager) CreateDaemonTask() (*apitask.Task, error) {
 	}
 
 	for _, mp := range dm.managedDaemon.GetMountPoints() {
-		err := mkdirAllAndChown(mp.SourceVolumeHostPath, daemonMountPermission, daemonUID, os.Getegid())
-		if err != nil {
+		if err := mkdirAllAndChown(mp.SourceVolumeHostPath, daemonMountPermission, daemonUID, os.Getegid()); err != nil {
 			return nil, err
 		}
 		var bindOptions = dockermount.BindOptions{}
 
 		if mp.PropagationShared {
 			// https://github.com/moby/moby/blob/master/api/types/mount/mount.go#L52
-			bindOptions.Propagation = dockermount.PropagationShared
+			bindOptions = dockermount.BindOptions{Propagation: dockermount.PropagationShared}
+			logger.Info(fmt.Sprintf("inside Propagation: %s", bindOptions.Propagation),
```

Comment:
"inside Propagation" reads like a debug log. How about we either make this a DEBUG level log or add a bit more context to the log if we want to keep it at INFO level?